doc/player_setup.md

```diff
@@ -186,7 +186,25 @@ It might be necessary to set the PulseAudio latency environment variable to 60 m
 
 ### AirPlay
 
-Snapserver supports [shairport-sync](https://github.com/mikebrady/shairport-sync) with the `stdout` backend and metadata support.
+Snapserver supports [shairport-sync](https://github.com/mikebrady/shairport-sync) with the `stdout` backend and metadata support. 
+These options are passed as parameters when Snapcast invokes Shairport-sync, so it _should_ work without modifying the shairport-sync.conf file.
+
+Some distros have shairport-sync packages in their main package repositories, and you may find that they 'just work', as long as you **disable the service**, so that Snapcast has full control of the shairport-sync process. Here is an example for some debian-based distros, but the need for sudo, the name of the package manager, and the syntax to install and control services may vary in your case
+
+```sh
+sudo apt install -y shairport-sync
+sudo systemctl disable --now shairport-sync
+```
+
+Here is an example configuration line for /etc/snapserver.conf but please see [the Snapcast Airplay configuration guide](configuration.md#airplay) for more details on the syntax and options.
+
+```plaintext
+source = airplay:///shairport-sync?name=Airplay[&devicename=Snapcast][&port=5000]
```

Comment:
The brackets are [EBNF notation](https://en.wikipedia.org/wiki/Extended_Backus%E2%80%93Naur_form) and simply mean that the value inside the brackets is optional. They (the brackets itself) should not appear in the configuration. 
If you omit those optional parameters, the default (Snapcast and 5000) will be used.
So please remove the brackets or the whole parameters.